test/xpu_api/numerics/complex.number/cmplx.over/proj.pass.cpp

```diff
@@ -22,15 +22,15 @@ template <class T>
 void
 test(T x, typename std::enable_if<std::is_integral<T>::value>::type* = 0)
 {
-    static_assert((std::is_same<decltype(dpl::proj(x)), dpl::complex<double> >::value), "");
+    STD_COMPLEX_TESTS_STATIC_ASSERT((std::is_same<decltype(dpl::proj(x)), dpl::complex<double> >::value), "");
```

Comment:
All tests with `decltype` should work independently of `constexpr`. So, all those lines can be used in `static_assert` expression.